classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- [ ] Let's kindly make sure a student is not able to be modify in this class. `attr_reader :students`